misk-admin/build.gradle.kts

```diff
@@ -34,6 +34,7 @@ dependencies {
   testImplementation(libs.kotlinTest)
   testImplementation(libs.wireRuntime)
   testImplementation(project(":misk-api"))
+  testImplementation(project(":misk-action-scopes"))
```

Comment:
Is this import necessary/correct?

Reply:
yep, some tests in misk-admin use classes from action-scope and the `testImplementation` just wasn't needed because there was an `api` dependency on actions-scopes which has been replaced with misk-api now.